awx/ui_next/src/screens/Host/HostDetail/HostDetail.jsx

```diff
@@ -68,4 +157,4 @@ HostDetail.propTypes = {
   host: Host.isRequired,
 };
 
-export default withI18n()(HostDetail);
+export default withI18n()(withRouter(HostDetail));
```

Comment:
Since we are trying to replace the withRouter HOC with react-router hooks in our functional components, please remove the HOC and retrieve the params and history object from the `useParams` and `useHistory` hooks.